test/test_yum_content_plugin.py

```diff
@@ -1,216 +0,0 @@
-# -*- coding: utf-8 -*-
-from __future__ import print_function, division, absolute_import
-
-import imp
-import os
-import types
-
-
-import mock
-from nose.plugins.skip import SkipTest
-
-try:
-    import yum
-except ImportError as e:
-    raise SkipTest(e)
-
-
-from . import fixture
-from . import stubs
-
-import subscription_manager.injection as inj
-from subscription_manager import productid
-
-# Yeah, this is weird. The yum plugins aren't on sys.path, nor are they in the
-# local src path that nosetest searches for modules. src/plugins is also not a
-# package dir (no __init__). And to top it off, the module name isn't a valid
-# python module name ('product-id.py', ie with an invalid '-').
-plugin_file_path = os.path.join(os.path.dirname(__file__), '../src/plugins/product-id.py')
-plugin_file = open(plugin_file_path, 'r')
-
-dir_path, module_name = os.path.split(plugin_file_path)
-module_name = module_name.split(".py")[0]
-
-
-# NOTE: the yum plugin 'product-id' gets imported as yum_product_id
-fp, pathname, description = imp.find_module(module_name, [dir_path])
-try:
-    yum_product_id = imp.load_module('yum_product_id', fp, pathname, description)
-finally:
-    fp.close()
-
-
-class TestYumPluginModule(fixture.SubManFixture):
-    def setUp(self):
-        super(TestYumPluginModule, self).setUp()
-
-    def test(self):
-        self.assertTrue(isinstance(yum_product_id, types.ModuleType))
-        return
-
-
-class TestYumProductManager(fixture.SubManFixture):
```

Comment:
Could it be possible to convert these unit tests to `test_dnf_content_plugin.py`, because `test_dnf_content_plugin.py` contains only one simple unit test and it seems that the most of logic of `product_id` and `subscription-manager` plugins is tested in this deleted module.

If it would be too complicate, then please add card for that.

Reply:
Yes, it would be nice to have more tests! I had a brief look at the dnf & librepo APIs, and they are (obviously) different from the yum API, so it is not as trivial as an almost 1:1 replacement.

Hence, I created a new card for this, ENT-4224.